CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 0.9.0 (October 8th, 2024)
+
+Features:
+* Add support for syncing [HVS rotating secrets](https://developer.hashicorp.com/hcp/docs/vault-secrets/auto-rotation): [GH-893](https://github.com/hashicorp/vault-secrets-operator/pull/893)
+* Add support for syncing [HVS dynamic secrets](https://developer.hashicorp.com/hcp/docs/vault-secrets/dynamic-secrets): [GH-917](https://github.com/hashicorp/vault-secrets-operator/pull/917) [GH-939](https://github.com/hashicorp/vault-secrets-operator/pull/939) [GH-934](https://github.com/hashicorp/vault-secrets-operator/pull/934) [GH-941](https://github.com/hashicorp/vault-secrets-operator/pull/941)
```

Comment:
Should we mention https://github.com/hashicorp/vault-secrets-operator/pull/889 in here somewhere?

Reply:
Added in d7fb9b38ba476b1ac7db65efb8be6bb23857428b